app/posts/find-teacher-training/2021-08-16-standardised-entry-requirement-descriptions.md

```diff
@@ -0,0 +1,67 @@
+
+For instance:
+
+> GCSE Maths, English and Science at Grade C or higher or equivalent
+
+> GCSEs: At least a Grade C or Grade 4 at GCSE (or equivalent) in English Language, Mathematics and Science is essential for all applicants.
+
+> GCSE in English (or approved equivalent) at grade C/Level 4 or above GCSE in Mathematics (or approved equivalent) at grade C/Level 4 or above GCSE in Science (or approved equivalent) at grade C/Level 4 or above
+
+These small differences may be confusing for candidates. The way providers had written requirements could also be difficult to understand.
+
+For the 2022 to 2023 cycle, we've changed the Publish interface to only ask providers to specify the entry requirements that may differ across courses.
+
+These are:
+
+* whether or not they [accept pending GCSEs and GCSE equivalency tests](/publish-teacher-training-courses/pending-gcses-equivalency-tests/)
+* the [minimum degree class required](/publish-teacher-training-courses/degree-entry-requirements/)
+* any additional degree subject requirements (for secondary courses)
+
+These changes allow us use consistent language to describe the standard entry requirements.
+
+## New content structure
+
+### Primary courses
+
+The standardised content is:
+
+> An undergraduate degree, or equivalent.
+>
+> GCSE grade 4 (&#8203;C) or above in English, maths and science, or equivalent qualification.
+
+Then the following paragraph is constructed from the provider’s answers to the pending GCSEs and equivalency tests questions, for example:
+
+> We’ll consider candidates who are currently taking GCSEs. We will not consider candidates who need to take GCSE equivalency tests.
+
+or
+
+> We’ll consider candidates who are currently taking GCSEs. We’ll consider candidates who need to take a GCSE equivalency test in English, maths, or science.
+
+Where providers do accept equivalency tests, they are asked to give details, and this free text field is shown at the end.
+
+### Secondary courses
+
+For secondary courses, a standardised paragraph is added about the degree subject:
+
+> Your degree subject should be in [course subject] or a similar subject. Otherwise you’ll need to prove your subject knowledge in some other way.
+
+Providers are able to add additional text after this if they have further degree subject requirements, such as a percentage of modules being in the relevant subject.
+
+The GCSE requirements for Secondary courses do not include science:
+
+> GCSE grade 4 (&#8203;C) or above in English and maths, or equivalent qualification.
+
+## Findings from user research
+
+There was confusion around the term ‘equivalent’, as it was for both ‘equivalency tests’, but also to mean an ‘equivalent’ qualification. To improve this, we’ve used the terms ‘GCSE equivalency test’ or ‘equivalent qualification’.
```

Comment:
The first line of this paragraph is still a bit clunky. Suggestion: 'There was confusion around the term 'equivalent', which refers to 'equivalency tests', but also 'equivalent' qualifications.'